### ISSUE TITLE
astro@next - terser issue

### ISSUE THREAD

**Issue by FredKSchott** [stackblitz/webcontainer-core#335]
### Describe the bug

The next version of Astro will be powered by Vite! This is a departure from our current Snowpack build engine.

I'm trying to get this running on Stackblitz so that I can include a link in our preview announcement blog post, but I'm seeing an odd terser bug. We don't run terser as far as I know, so I'm assuming that this is something happening in Stackblitz? "Error: Build failed with 1 error:" doesn't seem to be anywhere in our codebase (see below for full error).

### Link to the blitz that caused the error

https://stackblitz.com/github/snowpackjs/astro/tree/next/examples/minimal

### Steps to reproduce

1. Go to https://stackblitz.com/github/snowpackjs/astro/tree/next/examples/minimal
2. run `npm install astro@next--compiler` To install the new compiler
3. run `npm start` to start the Astro dev server
4. run `npm run build` to start the Astro build process

### Expected behavior

Both commands run locally. When you run `npm start`, Astro shows our "Welcome to Astro" minimal template page.

### Actual behavior

```
$ astro dev
 > node_modules/terser/dist/bundle.min.js:27827:39: error: Could not resolve "acorn" (mark it as external to exclude it from the bundle, or surround it with try/catch to handle the failure at run-time)
    27827 │                         return require("acorn").parse(files[name], {
          ╵                                        ~~~~~~~

Error: Build failed with 1 error:
node_modules/terser/dist/bundle.min.js:27827:39: error: Could not resolve "acorn" (mark it as external to exclude it from the bundle, or surround it with try/catch to handle the failure at run-time)
```

**Comment by FredKSchott** [comment]
One possible culprit:  Somewhere inside of Astro we do: 
```
import {rollupPluginHTML} from "@web/rollup-plugin-html";
```

`rollupPluginHTML` isn't ever used in the path of the dev server, but possibly just loading it is causing some issue inside of Stackblitz?

**Comment by jrvidal** [comment]
@FredKSchott  Thanks for the report, this does not seem to be a Stackblitz issue. I'm able to reproduce the problem in local as well:
* Download the code from the above blitz
* Run `npm install astro@next--compiler` (resolved to `0.0.0-compiler-2021824171345`)
* Run `npm start`

```
 @example/minimal@0.0.1 start
> astro dev

 > node_modules/terser/dist/bundle.min.js:27827:39: error: Could not resolve "acorn" (mark it as external to exclude it from the bundle, or surround it with try/catch to handle the failure at run-time)
    27827 │                         return require("acorn").parse(files[name], {
          ╵                                        ~~~~~~~

Error: Build failed with 1 error:
node_modules/terser/dist/bundle.min.js:27827:39: error: Could not resolve "acorn" (mark it as external to exclude it from the bundle, or surround it with try/catch to handle the failure at run-time)
```

Feel free to re-file if you find anything else.

**Comment by FredKSchott** [comment]
Ah shoot, no idea why this was working locally then. I'll follow up in our repo then. Thanks for the sanity check! 

Tracking: https://github.com/snowpackjs/astro/pull/1406#issuecomment-930506551
